src/ngraph/runtime/intelgpu/unit_test.manifest

```diff
@@ -0,0 +1 @@
+intelgpu_layouts
```

Comment:
If you want I can give you a complete list of all unit tests you need to exclude.